### PR TITLE
Reader: Add tracking for follow by url renders

### PR DESCRIPTION
**Pull request by blowery**
Adds render tracking for the follow by URL button in following manage.

### CODE REVIEW

File: client/reader/following-manage/index.jsx [hunk 1]
```diff
@@ -43,6 +43,29 @@ import { recordTrack } from 'reader/stats';
 
 const PAGE_SIZE = 4;
 
+function reportOnPropChange( Wrapped, prop ) {
```
Comment (samouri) [inline client/reader/following-manage/index.jsx]: we can get rid of this whole thing since its driven by a prop from following-manage/index!

Comment (bluefuton) [inline client/reader/following-manage/index.jsx]: Looks suspiciously reusable :) Shall we move it out to a separate file/lib?

**Comment by apeatling** [comment]
@blowery & @bluefuton Can we merge this? if not let's close it.

**Comment by blowery** [comment]
I don't recall why we wanted this. Closing for now.

**Pull request closed without merging**